File: 0136-single-number/0136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    
    var xors=0;
    
    for(let i=0;i<nums.length;i++){
  
        xors=xors^nums[i];
        
    }
    return xors;
    
    
};

File: 0136-single-number/0136-single-number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        unordered_map<int,int>mp;
        
        for(int i=0;i<nums.size();i++){
            mp[nums[i]]+=1;
        }
        int value=0;
        for(auto i:mp){
            
            if(i.second>1){
               continue;
            }
            else{
                value=i.first;
                break;
            
            }
        }
        
        return value;
    }
};